contracts/floaty/src/contract.rs

```diff
@@ -1,259 +1,58 @@
 use cosmwasm_std::{
-    entry_point, from_slice, to_binary, to_vec, AllBalanceResponse, BankMsg, Deps, DepsMut, Env,
-    Event, MessageInfo, QueryResponse, Response, StdError, StdResult,
+    entry_point, to_binary, Deps, DepsMut, Empty, Env, MessageInfo, QueryResponse, Response,
+    StdResult,
 };
+use rand_chacha::rand_core::SeedableRng;
 
-use crate::errors::HackError;
-use crate::msg::{ExecuteMsg, InstantiateMsg, QueryMsg, VerifierResponse};
-use crate::state::{State, CONFIG_KEY};
+#[cfg(target_arch = "wasm32")]
+use crate::instructions::run_instruction;
+use crate::{
+    instructions::{random_args_for, Value, FLOAT_INSTRUCTIONS},
+    msg::QueryMsg,
+};
 
 #[entry_point]
 pub fn instantiate(
-    deps: DepsMut,
+    _deps: DepsMut,
     _env: Env,
-    info: MessageInfo,
-    msg: InstantiateMsg,
-) -> Result<Response, HackError> {
-    deps.api.debug("here we go 🚀");
-
-    deps.storage.set(
-        CONFIG_KEY,
-        &to_vec(&State {
-            verifier: deps.api.addr_validate(&msg.verifier)?,
-            beneficiary: deps.api.addr_validate(&msg.beneficiary)?,
-            funder: info.sender,
-        })?,
-    );
-
-    // This adds some unrelated event attribute for testing purposes
-    Ok(Response::new().add_attribute("Let the", "hacking begin"))
+    _info: MessageInfo,
+    _msg: Empty,
+) -> Result<Response, String> {
+    Ok(Response::default())
 }
 
 #[entry_point]
 pub fn execute(
-    deps: DepsMut,
-    env: Env,
-    info: MessageInfo,
-    _msg: ExecuteMsg,
-) -> Result<Response, HackError> {
-    let data = deps
-        .storage
-        .get(CONFIG_KEY)
-        .ok_or_else(|| StdError::not_found("State"))?;
-    let state: State = from_slice(&data)?;
-
-    if info.sender == state.verifier {
-        let to_addr = state.beneficiary;
-        let balance = deps.querier.query_all_balances(env.contract.address)?;
-
-        let mut fl = balance[0].amount.u128() as f64;
-        fl *= 0.3;
-
-        let resp = Response::new()
-            .add_attribute("action", "release")
-            .add_attribute("destination", to_addr.clone())
-            .add_attribute("foo", fl.to_string())
-            .add_event(Event::new("hackatom").add_attribute("action", "release"))
-            .add_message(BankMsg::Send {
-                to_address: to_addr.into(),
-                amount: balance,
-            })
-            .set_data([0xF0, 0x0B, 0xAA]);
-        Ok(resp)
-    } else {
-        Err(HackError::Unauthorized {})
-    }
+    _deps: DepsMut,
+    _env: Env,
+    _info: MessageInfo,
+    _msg: Empty,
+) -> Result<Response, String> {
+    Ok(Response::default())
 }
```

Comment:
`execute` should be an optional entry point. Then you can remove this entirely. Could you double check in the VM where required exports are checked?

Reply:
Right, only instantiate is needed.